### ISSUE TITLE
How could I see the video in rig...

### ISSUE THREAD

**Issue by cww97** [vitchyr/rlkit#19]
I see

```
            save_video=True,
            save_video_period=100,
```

I can only see the png files in the folder `data`, I wanna know how or where can I find the video files.

**Comment by vitchyr** [comment]
It should be in the same folder inside of `data` as the pngs.

Note that you need to wait until the RL part of the algorithm is running. If you see output about "`train/epoch`", then it is still pretraining the VAE. Once you see the log outputting information about the "`Policy Loss`", then that means RL is running.

Lastly, `save_video_period=100` means that videos will only be logged once every 100 epochs. If you want to save videos more frequently, reduce this number.

**Comment by cww97** [comment]
> It should be in the same folder inside of `data` as the pngs.
> 
> Note that you need to wait until the RL part of the algorithm is running. If you see output about "`train/epoch`", then it is still pretraining the VAE. Once you see the log outputting information about the "`Policy Loss`", then that means RL is running.
> 
> Lastly, `save_video_period=100` means that videos will only be logged once every 100 epochs. If you want to save videos more frequently, reduce this number.

Thank you, sir. I can see the video for 100, 200, 300 now(due to my poor gpu, I need more time) 

In reality I have two Dobot Robotic Arms, and some cameras. I wonder where can I change the Robotic Arms and cameras in the sample experiment. And how could I test in real World.

Sorry it seem that my question is a little stupid, I did not learn a lot about Reinforcement Learning and Pytorch(even Machine Learning or Deep Learning). In another word, a rookie. I read the paper 'RIG', it seems that I can almost get the idea of what it means. Could you give me some advice of understand the algorithm, or DRL better ?
